tests/testsuite/freshness.rs

```diff
@@ -56,9 +56,19 @@ fn modifying_and_moving() {
     p.cargo("build")
         .with_status(101)
         .with_stderr_data(str![[r#"
-...
-[..]file not found[..]
-...
+[COMPILING] foo v0.0.1 ([ROOT]/foo)
+error[E0583]: file not found for module `a`
+ --> src/main.rs:1:1
+  |
+1 | mod a; fn main() {}
+  | ^^^^^^
+  |
+  = [HELP] to create the module `a`, create file "src/a.rs" or "src/a/mod.rs"
+  = [NOTE] if there is a `mod a` elsewhere in the crate already, import it with `use crate::...` instead
+
+For more information about this error, try `rustc --explain E0583`.
+[ERROR] could not compile `foo` (bin "foo") due to 1 previous error
```

Comment:
Ok, so this was for a compiler error.

For these we replace the bulk of the compiler message (the lines I have highlighted) with `...`

Reply:
Sure, but I think the L70 is [from Cargo](https://github.com/choznerol/cargo/blob/issue-14039-snapshot-port-freshness/src/cargo/core/compiler/mod.rs#L421-L425). I replaced L61-L69 with `...` in latest commit.